.ci-dockerfiles/llvm-7.0.0/Dockerfile

```diff
@@ -1,11 +1,12 @@
 FROM ponylang/ponyc-ci:ubuntu-14.04-base
 
-ENV LLVM_VERSION 4.0.1
+ENV LLVM_VERSION 7.0.0
 
 USER root
 
-RUN wget -O - http://llvm.org/releases/${LLVM_VERSION}/clang+llvm-${LLVM_VERSION}-x86_64-linux-gnu-debian8.tar.xz \
- | tar xJf - --strip-components 1 -C /usr/local/ clang+llvm-${LLVM_VERSION}-x86_64-linux-gnu-debian8
+RUN wget -O - http://llvm.org/releases/${LLVM_VERSION}/clang+llvm-${LLVM_VERSION}-x86_64-linux-gnu-ubuntu-14.04.tar.xz \
+ | tar xJf - --strip-components 1 -C /usr/local/ clang+llvm-${LLVM_VERSION}-x86_64-linux-gnu-ubuntu-14.04
+RUN chown -R root:root /usr/include
```

Comment:
The LLVM 7 installed files have  a really high uid that can't be remapped by circleci and causes the build to error out. This chown fixes the issue.